### ISSUE TITLE
Implement time frame filter in Grafana

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#44]
## User story
1. As a use
2. I want / need the data of a certain time frame
3. So that the data of the time frame is displayed

## Acceptance criteria
* The history of the data is stored in Prometeus.io
* The time frame is configurable in Grafana in certain steps or continous, e.g. last two days, or last week, or calender week 5
* Data of a certain time frame is displayed in Grafana

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated
